src/scene/morph.js

```diff
@@ -16,6 +16,7 @@ import { BUFFER_STATIC, TYPE_FLOAT32, SEMANTIC_ATTR15, ADDRESS_CLAMP_TO_EDGE, FI
  * @param {MorphTarget[]} targets - A list of morph targets.
  * @param {GraphicsDevice} graphicsDevice - The graphics device used to manage this morph target. If it is not provided, a device is obtained
  * from the {@link Application}.
+ * @property {MorphTarget} targets Gets an array of morph targets.
```

Comment:
This line can be removed if you accept my JSDoc change for `targets`. I made this change because it's the only way to make `targets` as read only in the JSDoc output. See [`GraphNode#children`](https://developer.playcanvas.com/en/api/pc.GraphNode.html#children) for an example.

Reply:
yep, I like this, done.